bootstrap-joining-demo/dns-api-mesos/build.sbt

```diff
@@ -5,7 +5,7 @@ version := "1.0"
 
 scalaVersion := "2.12.4"
 
-val akkaManagementVersion = "0.8.0"
+val akkaManagementVersion = "0.9.0"
```

Comment:
@ygree I have the same issue in my AWS EC2 demo since it's an independent project: every time a new version of akka management gets published , we basically have to update our demos :-( I propose we revert to a solution where our demos are actual sbt (sub)projects of the main project (the kubernetes api is set that way). The disadvantage is that it's harder for prospective users to copy-and-paste from the build.sbt file though .... @ygree what do you think ?

Reply:
Hmm, perhaps something like https://github.com/akka/akka-management/pull/120 could be a nice solution (rough draft)?